packages/client/src/views/DataProvider/QueryProvider.tsx

```diff
@@ -35,56 +23,3 @@ const QueryMapper = {
 export const getQueryMapping = (event: Event, action: Action) => {
```

Comment:
Consider moving these somewhere else as well if the component is not used anymore (this being a file for the component originally)